dotmotif/executors/NEUPRINT_JSON.py

```diff
@@ -0,0 +1,232 @@
+ATTRIBUTES = [
```

Comment:
Is there a way to dynamically fetch this list in the `NeuPrintExecutor` constructor?

Reply:
yes, I am using the `[fetch_all_rois](https://github.com/connectome-neuprint/neuprint-python/blob/ba13264e4cc91954b31ea002b679f482c14dfb57/neuprint/queries/rois.py#L8)` function from neuprint-python now.